pkg/apis/componentconfig/types.go

```diff
@@ -572,6 +572,7 @@ type KubeSchedulerConfiguration struct {
 	// HardPodAffinitySymmetricWeight represents the weight of implicit PreferredDuringScheduling affinity rule, in the range 0-100.
 	HardPodAffinitySymmetricWeight int
 	// Indicate the "all topologies" set for empty topologyKey when it's used for PreferredDuringScheduling pod anti-affinity.
+	// DEPRECATED: This is no longer used.
```

Comment:
I think you have to wait for @rrati's PR to know for sure. I believe based on the feedback from the kubernetes-dev thread he is going to do it like was done in #30468.